docs/examples/finetuning/llm_judge/finetune_llm_judge.ipynb

```diff
@@ -0,0 +1,1248 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #52.    def resolve_results(
</pre><p>could use some explanation on the logic here to make it easier to understand. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8596/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>